roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -4,6 +4,19 @@
     oc_bin: "{{openshift.common.client_binary}}"
     openshift_logging_namespace: "{{openshift_logging_namespace}}"
 
+- conditional_set_fact:
+    facts: "{{ openshift_logging_facts['elasticsearch']['configmaps']['logging-elasticsearch']['elasticsearch.yml'] | flatten_dict }}"
```

Comment:
Do we need to be concerned about the difference between an ops and non-ops deployment?